cron_job_definitions.go

```diff
@@ -85,3 +85,44 @@ func taskSyncTransactions(ctx context.Context, client *Client) error {
 	}
 	return err
 }
+
+func taskCalculateMetrics(ctx context.Context, client *Client) error {
+	m, enabled := client.Metrics()
+	if !enabled {
+		return errors.New("metrics are not enabled")
+	}
+
+	modelOpts := client.DefaultModelOptions()
+
+	if xpubsCount, err := getXPubsCount(ctx, nil, nil, modelOpts...); err != nil {
```

Comment:
I would love to create small private functions for each of those "if's", metrics :D 
But it's just a suggestion.

Reply:
Example:
```
func processXpubMetrics() {
// do sth
}
```

Reply:
I had several ideas about this but finally I decided to remain it as plain/simple as possible. For example - adding small private functions would break the "unwritten rule" that in this file there are only functions which are cron tasks. 

But if you find it beneficial - I will do as you proposed.